pxblue/react-native-showcase-demo
# 3267 - Upgrade to RN 70

Fixes BLUI-3267.


#### Changes proposed in this Pull Request:
- Upgrade to RN 70


#### Screenshots:


#### To Test:
- To be tested alongside the RN 70 upgrade for react-native-components
- `yarn start:showcase-ios`
- `yarn start:showcase-android`

> You will likely run into issues trying to get this running locally. Take a few minutes to google about and see if you can resolve your own build issues as it is likely to be an environment issue. The proper environment setup can be found on the official react-native docs: https://reactnative.dev/docs/environment-setup If you make it through the environment setup documentation and ensured you are setup properly reach out to me personally and I'll help you troubleshoot the remaining issues.
